airflow/utils/usage_data_collection.py

```diff
@@ -95,3 +96,15 @@ def get_executor() -> str:
 
 def get_python_version() -> str:
     return platform.python_version()
+
+
+def get_plugin_counts() -> dict[str, int]:
+    plugin_info = get_plugin_info()
+
+    return {
```

Comment:
Need to update docs too

Reply:
https://github.com/apache/airflow/blob/main/docs/apache-airflow/faq.rst#does-airflow-collect-any-telemetry-data

Reply:
Done.